src/status_im2/contexts/quo_preview/wallet/account_permissions.cljs

```diff
@@ -0,0 +1,72 @@
+(ns status-im2.contexts.quo-preview.wallet.account-permissions
+  (:require
+    [quo.core :as quo]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:key  :name
+    :type :text}
+   {:key  :address
+    :type :text}
+   {:key  :emoji
+    :type :text}
+   (preview/customization-color-option {:key :account-color})
+   {:key  :keycard?
+    :type :boolean}
+   {:key  :checked?
+    :type :boolean}
+   {:key  :disabled?
+    :type :boolean}
+   {:key     :token-details
+    :type    :select
+    :options [{:key :nil}
+              {:key :empty}
+              {:key :1-token}
+              {:key :3-tokens}
+              {:key :5-tokens}]}])
+
+(def token-details
+  {:nil      nil
+   :empty    []
+   :1-token  [{:token  "SNT"
+               :amount "100"}]
+   :3-tokens [{:token  "SNT"
+               :amount "100"}
+              {:token  "ETH"
+               :amount "18"}
+              {:token  "BTM"
+               :amount "1000"}]
+   :5-tokens [{:token  "SNT"
+               :amount "100"}
+              {:token  "ETH"
+               :amount "18"}
+              {:token  "BTM"
+               :amount "1000"}
+              {:token  "CFI"
+               :amount "1"}
+              {:token  "CK"
+               :amount "18"}]})
+
+(defn view
+  []
+  (let [state (reagent/atom {:name          "Trip to Vegas"
+                             :address       "0x2f0fbf0a93c5999e9b4410848403a02b38983eb2"
+                             :emoji         "😊"
+                             :account-color :blue
+                             :token-details :nil
+                             :keycard?      true
+                             :checked?      true
+                             :disabled?     false})]
+    (fn []
+      [preview/preview-container {:state state :descriptor descriptor}
+       [quo/account-permissions
+        {:account       {:name                (:name @state)
+                         :address             (:address @state)
+                         :emoji               (:emoji @state)
+                         :customization-color (:account-color @state)}
+         :token-details ((:token-details @state) token-details)
```

Comment:
**[Suggestion]** In cases where we see `((...` I think explicitly using `get` is clearer. I don't have references, but in Lisps I think it's preferable to avoid double or more parentheses on the left side because it's not as readable.

```clojure
;; Alternative 1
(get token-details (:token-details @state))

;; Alternative 2
(token-details (:token-details @state))
```

Alternative 2 is interesting and an idiomatic one, where we treat a map as a function. This is completely safe **when we know the map can't be nil**, often the case with static maps (such is the case for `token-details`).